### PR TITLE
Basic testing

### PR DESCRIPTION
**Pull request by MaximeBouton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ The NGSIM trajectory data is available in [our first release, with instructions
 You just clone it! Note that you also have to clone my [Vec](https://github.com/tawheeler/Vec.jl) package.
```
Comment (djp42) [inline README.md:15]:
```suggestion
The first step is to add the unregistered Julia package dependencies to your Julia environment:
```

File: README.md [hunk 2]
```diff
@@ -15,10 +15,10 @@ The NGSIM trajectory data is available in [our first release, with instructions
...
 
 ```julia
-Pkg.clone("https://github.com/sisl/Vec.jl.git")
-Pkg.clone("https://github.com/sisl/Records.jl.git")
-Pkg.clone("https://github.com/sisl/AutomotiveDrivingModels.jl.git")
-Pkg.clone("https://github.com/sisl/NGSIM.jl.git")
+Pkg.add(PackageSpec(url="https://github.com/sisl/Vec.jl"))
+Pkg.add(PackageSpec(url="https://github.com/sisl/Records.jl"))
+Pkg.add(PackageSpec(url="https://github.com/sisl/AutomotiveDrivingModels.jl"))
```
Comment (djp42) [inline README.md:21]: You added an AutoViz dependency to `runtests.jl`, should we add here too?

File: Project.toml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name = "NGSIM"
+uuid = "195157bc-cedd-5875-9ea7-7cfb75f30b3c"
+version = "0.0.0"
+
+[deps]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```
Comment (djp42) [inline Project.toml:8]: Should we add Project.toml dependencies for the unregistered packages too? I might have done that wrong in other repos if we aren't supposed to.

File: .travis.yml [hunk 4]
```diff
@@ -13,6 +13,11 @@ before_install:
   - sudo apt-get -qq update
 script: # use the default script section which is equivalent to the following
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-  - julia --check-bounds=yes -e 'using Pkg; Pkg.clone("https://github.com/tawheeler/Vec.jl.git"); Pkg.clone("https://github.com/sisl/Records.jl.git"); Pkg.clone("https://github.com/sisl/AutomotiveDrivingModels.jl.git"); Pkg.clone("https://github.com/sisl/AutoViz.jl.git"); Pkg.clone(pwd()); Pkg.build("NGSIM"); Pkg.test("NGSIM"; coverage=true)'
+  - julia --check-bounds=yes -e 'using Pkg; Pkg.add(PackageSpec(url="https://github.com/tawheeler/Vec.jl.git"));
```
Comment (djp42) [inline .travis.yml]:
```suggestion
  - julia --check-bounds=yes -e 'using Pkg; Pkg.add(PackageSpec(url="https://github.com/sisl/Vec.jl.git")); 
```

Comment (djp42) [reply .travis.yml]: Also, should all of these url's include the `.git`? I dont think it matters but I noticed you removed them from the install instructions.

File: Project.toml (hunk 3 above)
Comment (MaximeBouton) [reply Project.toml:8]: I could add them for consistency, I think it is better to have them here, but it forces to check in the Manifest.toml for travis to pass.

File: .travis.yml (hunk 4 above)
Comment (MaximeBouton) [reply .travis.yml]: It does not matter, I will remove it from travis

**Comment by MaximeBouton** [comment]
build is finally passing 🎉

**Review by djp42 — APPROVED** [review]

**Comment by djp42** [comment]
Awesome! It should be easier now to add more tests in the future.

**Review by raunakbh92 — APPROVED** [review]

**Pull request merged**